api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/BaseEncounterDao.java

```diff
@@ -111,6 +112,15 @@ protected void handleHasAndListParam(Criteria criteria, HasAndListParam hasAndLi
 									// No additional constraints needed, all Orders are given/assumed intent=order
 									handled = true;
 								}
+							} else if (MedicationRequest.SP_STATUS.equals(paramName)) {
+								// only supports ACTIVE at this time
+								if (paramValue != null) {
+									if (MedicationRequest.MedicationRequestStatus.ACTIVE.toString()
+									        .equals(paramValue.toUpperCase())) {
+										criteria.add(FhirDaoUtils.createActiveOrderCriterion("orders"));
+									}
+								}
+								handled = true;
```

Comment:
This may be bad practice, but note that I set "handled" if the paramName is matched, regardless of whether or not the paramValue is matched...  this allows me to add a query param like "&_has:MedicationRequest:encounter:status=&"  and interpret that as "any status" without logging a warning... as I'm writing this, I'm guessing this is likely bad practice , thoughts @ibacher ?

Reply:
Seems fine to me.